src/lasp_core.erl

```diff
@@ -701,7 +732,14 @@ map(Id, Function, AccId, Store, BindFun, ReadFun) ->
                                  lasp_orset ->
                                      lasp_orset:map(Function, V)
                              end,
-                  {ok, _} = BindFun(AccId, AccValue, Store)
+                  {ok, _} = BindFun(AccId, AccValue, Store);
+             %% A delta from the input will generate a new delta for the output
```

Comment:
When the `process()` receives a delta of the input from  the `ReadFun()`, the `lasp_orset:map()` transforms it to a delta of the output. And `bind()` will add the delta to the output. This is what I want to say in that comment. I will update it :)